app/src/test/kotlin/co/netguru/baby/monitor/client/feature/babycrynotification/NotifyBabyCryingUseCaseTest.kt

```diff
@@ -0,0 +1,36 @@
+package co.netguru.baby.monitor.client.feature.babycrynotification
+
+import co.netguru.baby.monitor.RxSchedulersOverrideRule
+import co.netguru.baby.monitor.client.feature.firebasenotification.FirebaseNotificationSender
+import co.netguru.baby.monitor.client.feature.firebasenotification.NotificationType
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.verify
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+class NotifyBabyCryingUseCaseTest {
+
+    @Rule
+    @JvmField
+    val schedulersRule = RxSchedulersOverrideRule()
+
+    private val notificationSender: FirebaseNotificationSender = mock()
```

Comment:
Suggestion:  Can be replaced with `private val notificationSender = mock<FirebaseNotificationSender>()`, but it's up to you.